Integrations/rasterize/rasterize.py

```diff
@@ -266,6 +292,7 @@ def rasterize_command():
     h = demisto.args().get('height', DEFAULT_H).rstrip('px')
     r_type = demisto.args().get('type', 'png')
     wait_time = int(demisto.args().get('wait_time', 0))
+    page_load = int(demisto.args().get('max_page_load_time', 0))
```

Comment:
```suggestion
    page_load = int(demisto.args().get('max_page_load_time', DEFAULT_PAGE_LOAD_TIME))
```
Think it is less confusing, because the zero here will be translated to DEFAULT_PAGE_LOAD_TIME in line 169.